__mocks__/vscode.ts

```diff
@@ -0,0 +1,11 @@
+const languages = {
+  registerDocumentFormattingEditProvider: jest.fn(),
+  registerDocumentRangeFormattingEditProvider: jest.fn()
+}
+
+const workspace = {}
+
+export {
+  languages,
+  workspace
+}

```

Comment:
I think we actually want the `vscode` module to be real. I think the `extensionHost` type launch config above may actually need to stay, such that it is a VSCode window instance running the Jest. I'm not sure what this looks like. Let's talk tomorrow and see if we can figure out what is needed.

Reply:
So far this is just running the unit tests, the extension launch tests shouldn't be (but are) affected.

Reply:
I do believe the unit tests run right now, however, I do not think `vscode` will work un-mocked and this would be important if we want to create real instances of VSCode features, like workspaces and trigger real events.